packages/perseus/src/widgets/image.tsx

```diff
@@ -191,7 +188,7 @@ class ImageWidget extends React.Component<Props> {
                 //
                 // TODO(david): If caption is only 1 line long, center-align
                 //     the text.
-                const alignment = this.props.alignment;
+                const alignment = "block";
                 const isImageFullWidth =
                     alignment === "block" || alignment === "full-width";
```

Comment:
Is this an intentional change? Are we no longer supporting an alignment option?

It looks like `image-editor.tsx` supports an alignment option. If we're removing `alignment` as an option I feel like that would require a lot more work and sign-offs from stakeholder - product / design / content.